doc/source/contributor.rst

```diff
@@ -0,0 +1,222 @@
+.. _developerdocumentation:
+
+=======================
+Developer Documentation
+=======================
+
+In this section, you will find documentation relevant to developing
+socok8s.
+
+.. _contributing:
+
+Contributor Guidelines
+======================
+
+Submodules
+----------
+
+This repository uses submodules. All of the following guidelines apply
+to this socok8s repository only. Please check other projects' practices
+before contributing to them, if a change needs to happen there.
+
+Before submitting code
+----------------------
+
+This is a very fast moving project, and you should probably contact us
+before starting to work on it.
+
+If you're willing to submit code, please remember the following rules:
+
+* All code should match our
+  :ref:`codeguidelines`.
+* All code requires to go through our :ref:`reviews`.
+* Documentation should be provided with the
+  code directly. See also :ref:`documentation`.
+
+.. _bug_reporting:
+
+Bug reporting process
+---------------------
+
+Bugs should be filed as Github issues.
```

Comment:
Hm ... shouldn't we actually involve Jira here? If there's a non-trivial bug, work on its fix should have Jira ticket assigned.

Reply:
Jira is internal, this code is public.

Reply:
If we are building a community, I would say JIRA should match our internal work related to a github issue.

If we are not, then I don't care, and we can replace with JIRA.

Keep in mind, JIRA is internal only.

Reply:
Yeah, you are right. I am not sure how to put together the need for outside feedback, community work and internal tracking. It might be right just to keep open tools here and solve the company part somewhere else